lib/tasks/sandbox_data.rake

```diff
@@ -17,13 +21,13 @@ namespace :lead_providers do
 
   desc "create NPQ seed schools and participants for API testing"
   task seed_schools_and_participants_npq: :environment do
-    logger = Logger.new($stdout)
-    logger.formatter = proc do |_severity, _datetime, _progname, msg|
-      "#{msg}\n"
-    end
+    return unless Rails.env.in?(%w[development review sandbox])
```

Comment:
could there be a reason for this type of logger formatting? maybe for logit/kibana

Reply:
Not sure.. seems code with no usage to me that's why I removed.. best practice is to use `Rails.logger` by default which level is can configured by environment.

Reply:
looks like it's very old :D